app/code/Magento/Customer/Test/Unit/Model/FileProcessorTest.php

```diff
@@ -313,6 +313,14 @@ public function testMoveTemporaryFileUnableToCreateDirectory()
 
         $destinationPath = 'customer/f/i';
 
+        $this->mediaDirectory->expects($this->at(0))
```

Comment:
Please extract this repeatable code to separate method

Reply:
Done